pkg/errors/errors.go

```diff
@@ -116,6 +117,8 @@ func GetErrorCode(err error) string {
 		return "Delete Failed"
 	} else if IsDeletePending(err) {
 		return "Delete Pending"
+	} else if IsWMIError(err) {
+		return "WMI Error"
```

Comment:
You need to return the WMI error text here. So you'll have to do "return err.Error()"